src/Migration/Adapter/Generator/PhinxMySqlGenerator.php

```diff
@@ -83,6 +83,10 @@ public function createMigration($name, $newSchema, $oldSchema): string
 
         $output = [];
         $output[] = '<?php';
+        if (!is_null($this->options['namespace'])) {
```

Comment:
This cuases warnings when the index is not set. Please use `if (!empty($this->options['namespace'])) {` instead.

Reply:
Done